modules/machine-learning-services/workspace/main.bicep

```diff
@@ -56,12 +56,10 @@ param tags object = {}
 @sys.description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
 param enableDefaultTelemetry bool = true
 
-// Identity
-@sys.description('Conditional. Enables system assigned managed identity on the resource. Required if `userAssignedIdentities` is not provided.')
-param systemAssignedIdentity bool = false
-
-@sys.description('Conditional. The ID(s) to assign to the resource. Required if `systemAssignedIdentity` is set to false.')
-param userAssignedIdentities object = {}
+@sys.description('Optional. The managed identity definition for this resource. At least one identity type is required.')
+param managedIdentities managedIdentitiesType = {
```

Comment:
Why the default?

Reply:
It's because at least one identity type is required for this resource type (the same for `disc-encryption-set`).